src/org/team1540/robot2017/Robot.java

```diff
@@ -27,15 +31,19 @@
  */
 public class Robot extends IterativeRobot {
 
-	public static final DriveTrain driveTrain = new DriveTrain();
+	//public static final DriveTrain driveTrain = new DriveTrain();
 	public static final Climber climber = new Climber();
-	public static final Feeder feeder = new Feeder();
-	public static final GearMechanism gearMechanism = new GearMechanism();
-	public static final Intake intake = new Intake();
-	public static final Shooter shooter = new Shooter();
+	//public static final Feeder feeder = new Feeder();
```

Comment:
Make sure to not comment out all of this code. When you merge, everything should work.